.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Please include the following information when opening an issue:
```

Comment:
But this will show up as an incomplete todo, so issues and PRs will have their status set by default as 0/5 steps done?